modules/telestion-example/src/main/java/de/wuespace/telestion/example/InformationReceiver.java

```diff
@@ -0,0 +1,28 @@
+package de.wuespace.telestion.example;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import de.wuespace.telestion.api.verticle.TelestionConfiguration;
+import de.wuespace.telestion.api.verticle.TelestionVerticle;
+import de.wuespace.telestion.api.verticle.trait.WithEventBus;
+import de.wuespace.telestion.example.information.SimpleInformation;
+import de.wuespace.telestion.example.messages.SimpleMessage;
+
+import java.util.Objects;
+
+public class InformationReceiver extends TelestionVerticle<InformationReceiver.Configuration> implements WithEventBus {
+	public record Configuration(@JsonProperty String inAddress) implements TelestionConfiguration {
+	}
```

Comment:
```suggestion
	public record Configuration(@JsonProperty String inAddress) implements TelestionConfiguration {  }
```

Sometimes a one liner is nice